earth_enterprise/src/fusion/autoingest/plugins.src/Database.src

```diff
@@ -228,17 +228,15 @@ void DatabaseAssetImplD::ConstructInputsFromRequest(
                             std::vector<SharedString>* inputs) {
   // make sure the project asset_ref_string's are valid
   if (!req.config.imageryProject.empty()) {
-    RasterProjectFactory::ValidateRefForInput(req.config.imageryProject,
-                                              AssetDefs::Imagery);
+    ValidateRefForInput<MutableDatabaseAssetD>(req.config.imageryProject, AssetDefs::Imagery, kDatabaseSubtype);
```

Comment:
Based on the contents of RasterProjectFactory, the template parameter should be RasterProjectAssetVersionD.

Reply:
There are likely other places like this where some of the parameters aren't correct. Would you take a pass through and make sure the parameters you're passing are the same as the ones used in the Factory that used to be called?